README.md

```diff
@@ -1,35 +1,30 @@
-# Membrane Template Plugin
+# Membrane Matroska Format
 
-[![Hex.pm](https://img.shields.io/hexpm/v/membrane_template_plugin.svg)](https://hex.pm/packages/membrane_template_plugin)
-[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_template_plugin)
-[![CircleCI](https://circleci.com/gh/membraneframework/membrane_template_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_template_plugin)
+[![Hex.pm](https://img.shields.io/hexpm/v/membrane_matroska_format.svg)](https://hex.pm/packages/membrane_matroska_format)
+[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_matroska_format)
+[![CircleCI](https://circleci.com/gh/membraneframework/membrane_matroska_format.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_matroska_format)
 
-This repository contains a template for new plugins.
-
-Check out different branches for other flavours of template.
+Membrane format describing Matroska container.
```

Comment:
I would take inspiration from https://github.com/membraneframework/membrane_rtp_format in this README